build.gradle

```diff
@@ -140,6 +152,7 @@ def versions = [
 ]
 
 dependencies {
+  compile group: 'org.jetbrains.kotlin', name:'kotlin-stdlib-jdk8'
```

Comment:
is this dependency required to be in application jar? perhaps just needed for development and can be marked as `compileOnly`

Reply:
yup .. its for the kotlin standard libraries, which includes a lot of useful extensions to java standard libraries.

Reply:
kotlin docs say use compile:
https://kotlinlang.org/docs/reference/using-gradle.html